sklearn/linear_model/tests/test_logistic.py

```diff
@@ -805,16 +802,24 @@ def test_logistic_regression_sample_weights():
                                n_classes=2, random_state=0)
     sample_weight = y + 1
 
+    def _set_cv_C(estimator):
+        # use smaller than the default number of Cs in LogisticRegressionCV
+        # for faster test time
+        if isinstance(estimator, LogisticRegressionCV):
+            estimator.set_params(Cs=3, cv=3)
+
     for LR in [LogisticRegression, LogisticRegressionCV]:
 
```

Comment:
How about here using
```py
kw = {'random_state': 42, 'fit_intercept': False, 'multi_class': 'ovr'}
if LR is LogisticRegressionCV:
    kw.update({'Cs': 3, 'cv': 3})
```

and construct below with **kw?

Reply:
Very good point, thanks.